rpcs3/Emu/CMakeLists.txt

```diff
@@ -63,6 +63,7 @@ target_sources(rpcs3_emu PRIVATE
 if(APPLE)
     target_sources(rpcs3_emu PRIVATE
         ../../darwin/util/sysinfo_darwin.mm
+        ../rpcs3/Input/GameController_pad_handler.mm
```

Comment:
you should put this into
../../darwin/input/GameController_pad_handler.mm

Reply:
Not with the other RPCS3 classes of a similar type?

Reply:
.mm files belong in the darwin folder

Reply:
Will do.